models/admin.js

```diff
@@ -1,13 +1,14 @@
 const { Schema, model } = require('mongoose')
 const {
-  enums: { ADMIN_STATUS_ENUM }
+  enums: { ROLE_ENUM }
 } = require('~/consts/validation')
 
 const adminSchema = new Schema({
   role: {
```

Comment:
Why do we need a role in the admin model?

Reply:
It will be used for role-based authentication